app/views/expenses/show.html.erb

```diff
@@ -0,0 +1,10 @@
+<p style="color: green"><%= notice %></p>
+
+<%= render @expense %>
+
+<div>
+  <%= link_to "Edit this expense", edit_expense_path(@expense) %> |
+  <%= link_to "Back to expenses", expenses_path %>
+
+  <%= button_to "Destroy this expense", @expense, method: :delete %>
+</div>
```

Comment:
Well done implementing the functionality for `edit` and `delete` transactions. 👏🏾 
This shows your willingness to go above and beyond the project requirements 💯

Reply:
Great to hear that my implementation of the edit and delete transaction functionalities was well received! Thank you for the positive feedback and for acknowledging my efforts to go above and beyond the project requirements. 😄